hudi-sync-common/pom.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
is this pom same as the current hudi-hive-sync pom?

Reply:
> Good with the module.. Few thoughts/suggestions
> 
> * Can we nest this under hudi-sync as `hudi-sync-common`, `hudi-sync-hive` and you can then introduce the aliyun metastore as `hudi-sync-aliyun`?
> * We need to just ensure there are no upgrades/backwards incompatible changes for existing users .

thanks ,
* i think "we nest this under hudi-sync as `hudi-sync-common`" is a good idea,i will modify like this
* "there are no upgrades/backwards incompatible changes for existing users ." i will add more tests to guarantee

Reply:
> is this pom same as the current hudi-hive-sync pom?
not exactly the same,i will minimize it